corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js

```diff
@@ -343,5 +343,14 @@ hqDefine("cloudcare/js/formplayer/utils/utils", function () {
         });
     };
 
+    Utils.getLastClassEl = function (classNames) {
```

Comment:
HQ uses a lot of jQuery, which could make this more concise. I think it'd be just be `$(classNames).last()`, which is short and readable enough that you wouldn't need a helper function.